scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPredictionRegression.scala

```diff
@@ -95,27 +87,4 @@ trait H2OMOJOPredictionRegression extends PredictionWithContributions {
     getWithLeafNodeAssignments() && H2OMOJOCache.canGenerateLeafNodeAssignments(
```

Comment:
@jakubhava why the condition for leaf node assignmets in regression problems is different from conditions in Binomial or Multinomial classification problems?

Reply:
Good question! That might be probably a bug.

Originally, the code had the condition as in binomial & multinomial. But our regression tests started to fail on GLM as it can't produce leaf node assignments (only tree-based models can) & I think I forgot to apply the change to other parts as well.

Let me fixed that in a followup PR

Reply:
AFIAK, contributions are not supported by GLM eighter. If the property is disabled by default, a user won't hit the problem anyway. I'm just wondering what it's better behavior, throw an exception or silently ignore the configuration of the property. An exception might not be a bad solution. WDYT?

Reply:
Created https://github.com/h2oai/sparkling-water/pull/2218, let's continue there :)

Regarding the exception -> I think that a user warning would do in this case, WDYT?

Reply:
Yes, warning is fine.